content/logs/languages/csharp.md

```diff
@@ -271,9 +271,42 @@ logs:
 
 That's it! Now, all your logs are going to be in proper JSON automatically understood by your Datadog application.
 
+## Agentless logging
+
+It is possible to stream logs from your application to Datadog or to the Datadog Agent directly. This is not the recommended setup as handling connection issues should not be done directly in your application, but it might not be possible to log into a file when your application is running on a machine that cannot be accessed.
+
+### SeriLog 
+
+Install Datadog Serilog sink that send events and logs to Datadog. By default the sink uses a TCP connection over SSL.
```

Comment:
Install the Datadog Serilog sink, which send events and logs to Datadog. By default the sink uses a TCP connection over SSL.